src/test/container-features/featuresCLICommands.test.ts

```diff
@@ -66,14 +66,39 @@ describe('CLI features subcommands', async function () {
 
 			const expectedTestReport = `  ================== TEST REPORT ==================
 ✅ Passed:      'util'
-✅ Passed:      'some_scenario'`;
+✅ Passed:      'some_scenario'
+✅ Passed:      'some_scenario_2'
+✅ Passed:      'random_scenario'`;
 			const hasExpectedTestReport = result.stdout.includes(expectedTestReport);
 			assert.isTrue(hasExpectedTestReport);
 
 			assert.isTrue(result.stderr.includes('I AM A DIFFERENT SCRIPT'));
 			assert.isTrue(result.stderr.includes('I AM A HELPER SCRIPT FOR A SCENARIO'));
 		});
 
+		it('succeeds when passing --filter some_scenario', async function () {
+			const collectionFolder = `${__dirname}/example-v2-features-sets/sharing-test-scripts`;
+			let success = false;
+			let result: ExecResult | undefined = undefined;
+			try {
+				result = await shellExec(`${cli} features test --project-folder ${collectionFolder} --base-image mcr.microsoft.com/devcontainers/base:ubuntu --log-level trace`);
```

Comment:
Not sure if I'm missing something, but shouldn't the command contain `--filter` flag? 🤔